test/system/announcements_test.rb

```diff
@@ -125,17 +125,17 @@ class AnnouncementsTest < ApplicationSystemTestCase
 
     assert has_no_button? '公開'
     click_button '作成'
-    assert_text 'お知らせを作成しました'
+    assert page.has_css?('p.flash__message', text: 'お知らせを作成しました')
 
     visit_with_auth '/notifications', 'hatsuno'
     assert_text 'お知らせ「タイトルtest」'
 
     visit_with_auth '/announcements', 'komagata'
     click_on 'タイトルtest'
-    accept_confirm do
+    page.accept_confirm do
```

Comment:
```suggestion
   accept_confirm do
```

pageは書かなくてもいいように思います〜

Reply:
こちら修正しました！
https://github.com/fjordllc/bootcamp/pull/6212/commits/98d07156f81bbfbfe7773a2a79d010d7a125ebe6